Docs: how to use pgAdmin (#3915)
cal-itp/benefits: docs/guides/postgresql.md

1. Trigger the upgrade in Azure
1. Update `azurerm_postgresql_flexible_server` in [terraform][] and `terraform apply` to avoid reversion

## Use pgAdmin

> You can use pgAdmin to manage the PostgreSQL server via a GUI. Setting up pgAdmin is slightly different depending on the environment where it is being hosted.

Reviewer: File this suggestion under 'take it or leave it' 😛 
```suggestion
pgAdmin provides a GUI to manage the PostgreSQL server. pgAdmin is setup slightly differently depending on the hosted environment.
```

### Local development environment

If this is your first time using pgAdmin or if you have recently cleared your associated Docker volume (`pgadmin_data`), you will need to register the PostgreSQL server:

1. Select _Add New Server_ from the dashboard or navigate to _Object > Register > Server_.
2. Under the _General_ tab, provide any descriptive _Name_ for the server.

> 3. Under the _Connection_ tab, configure the following properties based on your environment variables:

Reviewer: i think it'd be worthwhile to be a little bit more explicit about _where_ the configuration happens.

```suggestion
1. Navigate to http://localhost:8081
2. Select _Add New Server_ from the dashboard or navigate to _Object > Register > Server_.
3. Under the _General_ tab, provide any descriptive _Name_ for the server.
4. Under the _Connection_ tab, configure the following properties based on your environment variables:
```

   - _Host name/address_: Value of `POSTGRES_HOSTNAME` (usually `postgres`)
   - _Port_: Value of `POSTGRES_PORT` (usually `5432`)
   - _Maintenance database_: Value of `POSTGRES_DB` (usually `postgres`)
   - _Username_: Value of `POSTGRES_USER` (usually `postgres`)
   - _Password_: Value of `POSTGRES_PASSWORD` (usually `postgres`)
   - _Save password?_: Yes

Note that because `PGADMIN_CONFIG_SERVER_MODE=false`, pgAdmin runs in desktop mode which uses an automatic default login. After setting it up, you can start using pgAdmin to manage the database, e.g. going to _Servers > Name > Databases > django > Schemas > public > Tables_ to view the application's tables.

### Cloud environments

> To use pgAdmin in any of the Cloud environments:

Reviewer: ```suggestion
To use pgAdmin in any of the cloud environments:
```

1. Select the _pgAdmin Container App_ resource in Azure Portal.
2. Go to _Networking > Ingress > IP Restrictions_ and add your local public IP as an allowed _Source_.

> 3. Open pgAdmin by going to the Container App's _Application Url_.

Reviewer: ```suggestion
3. Open pgAdmin by launching the Container App's _Application Url_.
```
